apis/v1alpha2/httproute_types.go

```diff
@@ -722,9 +722,7 @@ type HTTPRequestMirrorFilter struct {
 	//
 	// Support: Extended for Kubernetes Service
 	// Support: Custom for any other resource
-	//
-	// +optional
-	BackendRef *BackendObjectReference `json:"backendRef,omitempty"`
+	BackendRef BackendObjectReference `json:"backendRef"`
```

Comment:
CI failing because tests are not compiling.

Reply:
thanks, should be good now